api/src/main/java/com/github/retrooper/packetevents/wrapper/play/client/WrapperPlayClientSetDifficulty.java

```diff
@@ -37,17 +37,17 @@ public WrapperPlayClientSetDifficulty(Difficulty difficulty) {
 
     @Override
     public void read() {
-        this.difficulty = Difficulty.getById(readByte());
+        this.difficulty = Difficulty.getById(readUnsignedByte());
```

Comment:
did you verify that unsignedByte is the correct thing here?

Reply:
yea